zoo_frontend/src/components/Header.jsx

```diff
@@ -76,13 +79,6 @@ const Header = ({ classes, drawerOpen, handleDrawerOpen }) => (
               </Typography>
             </Link>
             <div className={classes.grow} />
-            {hasAccess(account.role, Admin.roles) &&
-              <Link href={Admin.link}>
-                <Button variant="contained" className={classes.button} color="secondary">
-                        Admin
-                </Button>
-              </Link>
-            }
             <Typography style={{ paddingLeft: '10px', paddingRight: '10px' }}>
               Hello {account.loggedIn ? `${account.firstName} ${account.lastName}` : 'Guest'}!
             </Typography>
```

Comment:
line 85 - change to `{ account.loggedIn &&`
This fixed issue on my local branch since localStorage vars like .role didn't get reset when first launching the app in dev state or possibly prod states. Had no side effects using loggedIn instead.